docs/get-started/getting-started-databricks.md

```diff
@@ -47,7 +47,19 @@ when using the plugin. Queries may still see significant speedups even with AQE
    (where N is the number of GPUs per node). This will result in failed executors when starting the
    cluster.
 
-5. Databricks makes changes to the runtime without notification.
+5. Parquet rebase mode is set to "LEGACY" by default.
+
+   Below 2 rebase mode related Spark configurations are set to `LEGACY` by default on Databricks:
+   
+   ```
+   spark.sql.legacy.parquet.datetimeRebaseModeInWrite
+   spark.sql.legacy.parquet.int96RebaseModeInWrite
+   ```
+   
+   It will cause CPU fallback when writing parquet files. If you do not need "LEGACY" mode, please 
+   set them back to "EXCEPTION" which is the default value in Apache Spark.
```

Comment:
```suggestion
   These settings will cause a CPU fallback for Parquet writes involving dates and timestamps.
   If you do not need `LEGACY` write semantics, set these configs to `EXCEPTION` which is
   the default value in Apache Spark.
```

Reply:
Can we say that this is the default value in Apache Spark 3.0 and higher?

Reply:
> Can we say that this is the default value in Apache Spark 3.0 and higher?

Added.